modules/apps/29-fee/keeper/grpc_query.go

```diff
@@ -63,3 +64,41 @@ func (k Keeper) IncentivizedPacket(c context.Context, req *types.QueryIncentiviz
 		IncentivizedPacket: &fee,
 	}, nil
 }
+
+// IncentivizedPacketsForChannel implements the IncentivizedPacketsForChannel gRPC method
+func (k Keeper) IncentivizedPacketsForChannel(c context.Context, req *types.QueryIncentivizedPacketsForChannelRequest) (*types.QueryIncentivizedPacketsForChannelResponse, error) {
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "empty request")
+	}
+
+	ctx := sdk.UnwrapSDKContext(c).WithBlockHeight(int64(req.QueryHeight))
+
+	var packets []*types.IdentifiedPacketFees
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), []byte(string(types.KeyFeesInEscrowChannelPrefix(req.PortId, req.ChannelId))+"/"))
+	_, err := query.Paginate(store, req.Pagination, func(key, value []byte) error {
+		// the key returned only includes the sequence
+		seq, err := strconv.ParseUint(string(key), 10, 64)
+		if err != nil {
+			return err
+		}
```

Comment:
left off a test case because then I'd have to set an invalid key. I could do this, but the correctness of keeper setting shouldn't really be tested by grpc query tests